packages/core/src/shared/extensionGlobals.ts

```diff
@@ -160,6 +160,10 @@ export function isWeb() {
     return globals.isWeb
 }
 
+export function isEmbedded() {
+    return globals.isEmbedded
```

Comment:
can callers use `globals.isEmbedded` instead? It's not really a goal to have module-level aliases to all of the `globals` fields.

Reply:
Sure. I was just following the pattern for `isWeb`, but if it's preferred to access globals directly, I will make that change.

Reply:
isWeb is a special case rather than a pattern